editing/edit-feature-attachments/src/main/java/com/esri/samples/edit_feature_attachments/EditFeatureAttachmentsSample.java

```diff
@@ -113,13 +119,16 @@ public void start(Stage stage) {
       // add controls to the panel
       controlsVBox.getChildren().addAll(addAttachmentButton, deleteAttachmentButton, attachmentsLabel, attachmentList);
 
-      // create a map view
-      mapView = new MapView();
+      // create a map with streets basemap style
```

Comment:
Another one? (I won't comment on the rest)
```suggestion
      // create a map with the streets basemap style
```

Reply:
Have been through all changed samples and made this tidy up